CMakeLists.txt

```diff
@@ -17,7 +17,8 @@
 # under the License.
 #
 
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
When raising cmake or other requirements, it needs to be documented in the README.

Reply:
updated in https://github.com/apache/nifi-minifi-cpp/pull/1098/commits/b20daf04ede9c0d02c1934b75a00da6254cc2e1f